src/components/Review.jsx

```diff
@@ -8,12 +8,19 @@ function Review() {
   const { review_id } = useParams();
   const [review, setReview] = useState({});
   const [isLoading, setIsLoading] = useState(true);
+  const [disabled, setDisabled] = useState(false);
 
   const formatDate = (dataString) => {
     const date = new Date(dataString);
     const formattedDate = date.toLocaleString();
     return formattedDate;
   };
+  const handleClick = () => {
+    api.patchReviewVotes(review_id).then((updatedReview) => {
```

Comment:
using optimistic rendering will improve UX
- removing the delay
- ensuring that the given like is accounted for, as opposed to including concurrent likes from elsewhere